app/helpers/application_helper.rb

```diff
@@ -141,7 +141,7 @@ def format_phone_number(number)
     [number[0..3], number[4..6], number[7..10]].join(' ')
   end
 
-  def display_primary_navigation?
-    @primary_navigation_disabled != true
+  def yes_no_options
+    [%w[Yes true], %w[No false]]
```

Comment:
**not a blocker** - could apply translations in the dictionary and use these

Reply:
I've thought about it and will probably change it in the future to do that 👍